homeassistant/components/airnow/strings.json

```
@@ -25,6 +25,9 @@
     "sensor": {
       "o3": {
         "name": "[%key:component::sensor::entity_component::ozone::name%]"
+      },
+      "station": {
+        "name": "PM2.5 Reporting Station"
```

Isn't this the reporting station for every data point?
In any case
```suggestion
        "name": "PM2.5 reporting station"
```

---

The API returns a reporting station associated with each measurand (AQI, PM2.5, and ozone). Right now the integration just stores the reporting station associated with the PM2.5 measurement. It's not clear if there would ever be a case where different reporting stations are used for some measurands, but I wanted to be clear where the data is coming from.

---

Understood! If you could still look at the capital letters :)